pkg/plugins/platform/local.go

```diff
@@ -186,6 +185,24 @@ func (l *LocalApp) ImportDB(imPath string) error {
 	return nil
 }
 
+// SiteStatus returns the current status of an application based on the web container.
+func (l *LocalApp) SiteStatus() string {
+	webContainer, err := l.FindContainerByType("web")
+	if err != nil {
+		return "not found"
+	}
+
+	status := util.GetContainerHealth(webContainer)
+	if status == "exited" {
+		return "stopped"
+	}
+	if status == "healthy" {
+		return "running"
+	}
```

Comment:
Don't we need restarting or other options here? (If we get to 3 options it's nicer to do a switch statement.)  

There are a couple of problems with this SiteStatus() function that we already know about:
* It doesn't take into account other containers (just web)
* It's an explicit translation approach from container->site, which may not be adequate.